phoebe/frontend/default_bundles/default_contact_binary.bundle

```diff
@@ -144,7 +144,7 @@ null
 "kind": "star",
 "context": "component",
 "description": "Critical (maximum) value of the equivalent radius for the given morphology",
-"value": 1.6724563972838287,
+"value": 1.6724563972838289,
```

Comment:
I vaguely remember seeing updates here recently... do we understand why this is changing?